pkg/ottl/contexts/ottllog/log.go

```diff
@@ -55,6 +73,22 @@ func (tCtx TransformContext) getCache() pcommon.Map {
 	return tCtx.cache
 }
 
+func (tCtx TransformContext) getScopeLogs() plog.ScopeLogs {
+	return tCtx.scopeLogs
+}
+
+func (tCtx TransformContext) getResourceLogs() plog.ResourceLogs {
+	return tCtx.resourceLogs
+}
+
+func (tCtx TransformContext) GetScopeSchemaURLItem() internal.SchemaURLItem {
+	return tCtx.logSchemaURLItem
+}
+
+func (tCtx TransformContext) SetScopeSchemaURLItem(schema internal.SchemaURLItem) {
+	tCtx.logSchemaURLItem = schema
+}
```

Comment:
@evan-bradley I am bit unsure about this, because the GetResourceSchemaURLItem, would directly give access to the entire scopeLogs object, which does not follow the the interface of internal.SchemaURLItem.

Reply:
I think the way you have it looks good now, thanks.